packages/commonwealth/client/scripts/views/pages/NotificationSettings/useSupportsPushNotifications.ts

```diff
@@ -0,0 +1,15 @@
+import { getBrowserType } from 'helpers/browser';
+import useAppStatus from 'hooks/useAppStatus';
+
+export function useSupportsPushNotifications() {
```

Comment:
per convention we use, let's change it to const, so we keep consistency 
`export const useSupportsPushNotifications = () => {}`